cpu/cc2538/dev/cc2538-rf.c

```diff
@@ -120,6 +120,17 @@ static const uint8_t magic[] = { 0x53, 0x6E, 0x69, 0x66 };      /** Snif */
 #define CC2538_RF_AUTOACK 1
 #endif
 /*---------------------------------------------------------------------------*/
+#if CC2538_RF_CONF_SFD_TIMESTAMPS
+static rtimer_clock_t volatile cc2538_sfd_rtime;
+#endif
+
+/* Are we currently in poll mode? Disabled by default */
+static uint8_t volatile poll_mode = 0;
+/* Do we perform a CCA before sending? Enabled by default. */
+static uint8_t send_on_cca = 1;
```

Comment:
`send_on_cca` seems unused, i.e. always disabled. That's fine for TSCH. But then no need for the variable. Simply always return 0 in `get_option`, and success for `set_option` if set to 0, failure if set to 1.